src/test/scala/fr/davit/capturl/parsers/PathParserSpec.scala

```diff
@@ -22,6 +22,10 @@ class PathParserSpec extends FlatSpec with Matchers {
     parse("directory/?query") shouldBe Path.Segment("directory", Slash())                                -> "?query"
     parse("/one//path?query") shouldBe Slash(Segment("one", Slash(Segment("", Slash(Segment("path")))))) -> "?query"
 
+    // path with empty query
+    parse("path?") shouldBe Segment("path")                                                              -> "?"
```

Comment:
`PathParser` isn't used for the delimiter in lazy Iri

Reply:
I'm aware - should I delete the test?

Reply:
I deleted the test case.